TundraTD/Assets/Scripts/Level/TickConvertation.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Level
+{
+    static class TickConvertation
```

Comment:
I suggest you rename this class
```suggestion
    static class TickConverter
```